cms/plugin_base.py

```diff
@@ -219,6 +219,22 @@ def get_cache_expiration(self, request, instance, placeholder):
         """
         return None
 
+    def get_vary_cache_on(self, request, instance, placeholder):
+        """
+        Provides hints to the placeholder, and in turn to the page for
+        determining VARY headers for the response.
+
+        Must return one of:
+            - None (default),
+            - String of a case-sensitive header name, or
```

Comment:
While it's nice to return a string, I think we should stick to lists or tuples (or other no string iterables).
I think supporting both just adds extra overhead, developers can just return ['some-header']